### ISSUE TITLE
Add an ungrouping option for the apps and more

### ISSUE THREAD

**Issue by udobehm** [jetspiking/Taskbar11#22]
Hi,

you mind adding a function to ungroup apps to your Taskbar11 app? Would like them to appear as it was possible in Windows 10 when the grouping function was deselected.

Also could you add the Windows 10 flyout menus as the StartAllBack app offer them to be shown? I'm talking of the old wifi, audio and energy symbols and functions as Windows 10 was offering them and as you can configure them in the StartAllBack app.

Hope you may implement these functions, mostly the ungrouping function that I desparately need...

Thank you!

**Comment by jetspiking** [comment]
Hi there!

Taskbar11 currently only modifies existing UI-layout by modifying registry keys. Currently no extra functionalities can be added if Microsoft does not have an implementation for these.

However, I am currently rewriting Taskbar11 and I hope to bring more functionalities soon. For unsupported options I am experimenting with a toolbar, which should contain missing features.

**Comment by udobehm** [comment]
Sounds good.

Hope you may implement these functionalities...

Von: Dustin Hendriks ***@***.***>
Gesendet: Donnerstag, 13. Januar 2022 11:29
An: jetspiking/Taskbar11 ***@***.***>
Cc: udobehm ***@***.***>; Author ***@***.***>
Betreff: Re: [jetspiking/Taskbar11] Add an ungrouping option for the apps and more (Issue #22)


Hi there!

Taskbar11 currently only modifies existing UI-layout by modifying registry keys. Currently no extra functionalities can be added if Microsoft does not have an implementation for these.

However, I am currently rewriting Taskbar11 and I hope to bring more functionalities soon. For unsupported options I am experimenting with a toolbar, which should contain missing features.

-
Reply to this email directly, view it on GitHub<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjetspiking%2FTaskbar11%2Fissues%2F22%23issuecomment-1012001102&data=04%7C01%7C%7C23867cfa5a524e5ef34b08d9d67f7a5d%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637776665261163554%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=dI7k8%2Fl1uF9kKOwERbM5J%2BpYV4E6TFT%2BvdrKY3lJBq0%3D&reserved=0>, or unsubscribe<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAMECQDH4TGEB5U6S3CZJZUTUV2SNZANCNFSM5LZ6HARQ&data=04%7C01%7C%7C23867cfa5a524e5ef34b08d9d67f7a5d%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637776665261163554%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=dU%2FbtiBiNkYs0YtjUtW%2Fp%2FzVBVoptRy0QryamObJtX8%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7C23867cfa5a524e5ef34b08d9d67f7a5d%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637776665261163554%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Kwzua8TiuAWCjcGl0x%2FTgHL%2BUFW9ToUL0%2Bsmk7hCY0A%3D&reserved=0> or Android<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7C23867cfa5a524e5ef34b08d9d67f7a5d%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637776665261163554%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=fLOY%2FihkRXGwJsmAhsYGxQr7UI2b3Vm04UuH1mP0akI%3D&reserved=0>.

**Comment by jetspiking** [comment]
Hi there!

Taskbar11 was updated with a toolbar and the user interface has been changed. It does not implement all your requested features at the moment. However, you can search and launch applications from the toolbar.

**Comment by udobehm** [comment]
Ok, thanks. As far as I can see none of my desired features are currently available. Do I see that right?

**Comment by jetspiking** [comment]
That is correct. The Volume, Wi-Fi and Energy-controls could be implemented in the next version. I'm still thinking about to implement a way to switch between already opened apps. However, I am attempting to prevent extra windows, which makes the UI design a bit more challenging.

**Comment by udobehm** [comment]
Ok. The ungrouping function would be most important to me and as far as I can see to a lot more users too. Was perfect if you could implement that functionality...
________________________________
